internal/provider/harness_docker_resource.go

```diff
@@ -0,0 +1,316 @@
+package provider
+
+import (
+	"context"
+	"fmt"
+	"path/filepath"
+
+	"github.com/chainguard-dev/terraform-provider-imagetest/internal/containers/provider"
+	"github.com/chainguard-dev/terraform-provider-imagetest/internal/harnesses/container"
+	"github.com/chainguard-dev/terraform-provider-imagetest/internal/harnesses/docker"
+	"github.com/docker/docker/api/types/mount"
+	"github.com/google/go-containerregistry/pkg/name"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/booldefault"
+
+	"github.com/chainguard-dev/terraform-provider-imagetest/internal/log"
+	"github.com/hashicorp/terraform-plugin-framework/path"
+	"github.com/hashicorp/terraform-plugin-framework/resource"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/stringdefault"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+)
+
+const (
+	ContainerImage = "cgr.dev/chainguard/wolfi-base:latest" // TODO(mauren): create a docker-cli image
```

Comment:
```suggestion
	ContainerImage = "cgr.dev/chainguard/docker-cli:latest"
```

🔥 🎉

Reply:
I'll make the change in a follow-up if that's alright? Just want to test and make sure everything will be correct.